lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -74,7 +74,7 @@ def to_predicate_matcher(name)
             name[0..-2]
           when 'exist?', 'exists?'
             'exist'
-          when /^has_/
+          when /\Ahas_/
```

Comment:
For example, this is looking for method names starting with `has_`. Rather than using `^`, which is a multi-line match, `\A` seems more appropriate.